bindings/gumjs/gumscript-runtime-objc.js

```diff
@@ -536,7 +594,7 @@
                         }
                     }
 
-                    return cachedMethodNames;
+                    return Object.keys(objCObjectBuiltins).concat(cachedMethodNames);
```

Comment:
Pedantically we shouldn't return all the builtins here; we should only return the properties and not things on the prototype.